_posts/components/2018-09-24-comments.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title:  Comments
+category: components
+description: The comments section for a `Commentable` model.
+---
+
+.card.mb-3
+  %h5.card-header
+    Comments for package123 (2)
+  .card-body#comments
+    //= render partial: 'webui2/webui/comment/show', locals: { commentable: @package }
```

Comment:
This is not about having a copy & paste database. You should include the code from the partial IMHO.

Reply:
I wasn't sure how to proceed. I will keep the comment, but add the output of this partial.